generated/nidaqmx/_library_interpreter.py

```diff
@@ -2332,8 +2332,7 @@ def get_buffer_attribute_uint32(self, task, attribute):
         cfunc = lib_importer.cdll.DAQmxGetBufferAttribute
         with cfunc.arglock:
             cfunc.argtypes = [
-                lib_importer.task_handle, ctypes.c_int32,
-                ctypes.POINTER(ctypes.c_uint32)]
+                lib_importer.task_handle, ctypes.c_int32]
```

Comment:
You also need to update the template.

Attribute get/set/reset functions should use double-checked locking like normal functions:
```
        if cfunc.argtypes is None:
            with cfunc.arglock:
                if cfunc.argtypes is None:
                    cfunc.argtypes = [...]
```

Non-attribute varargs functions like `get_analog_power_up_states` should still hold the lock when assigning `cfunc.argtypes` and calling `cfunc`, as they do now. I guess those are in a different template, though.